tools/importer-rest-api-specs/components/parser/resourceids/generate_names.go

```diff
@@ -12,11 +12,11 @@ import (
 	"github.com/hashicorp/pandora/tools/importer-rest-api-specs/models"
 )
 
-func (p *Parser) generateNamesForResourceIds(input []models.ParsedResourceId) (*map[string]models.ParsedResourceId, error) {
-	return generateNamesForResourceIds(input, p.logger)
+func (p *Parser) generateNamesForResourceIds(input []models.ParsedResourceId, uri2ResourceId map[string]ParsedOperation) (*map[string]models.ParsedResourceId, error) {
```

Comment:
Can we rename this to 
```suggestion
func (p *Parser) generateNamesForResourceIds(input []models.ParsedResourceId, uriToResourceId map[string]ParsedOperation) (*map[string]models.ParsedResourceId, error) {
```

Reply:
@wuxu92 looks like this one's still pending?

Reply:
sorry i missed this one. updated jus now.